docs/velox-backend-support-progress.md

```diff
@@ -202,7 +208,7 @@ Gluten supports 199 functions. (Draw to right to see all data types)
 | regexp_like                   | regexp_like            | rlike                 | S      | Not support lookaround |         |      |       |     |      |       |        |      |           | S      |         |      |        |          |       |      |        |      |
 | regexp_replace                | regexp_replace         |                       | S      |                        |         |      |       |     |      |       |        |      |           | S      |         |      |        |          |       |      |        |      |
 | rlike                         |                        | rlike                 | S      | Not support lookaround |         |      |       |     |      |       |        |      |           | S      |         |      |        |          |       |      |        |      |
```

Comment:
`lookaround not supported`?

Reply:
@yma11, yes, this is a known limit of RE2.